src/foam/comics/v3/DetailView.js

```diff
@@ -0,0 +1,398 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.comics.v3',
+  name: 'DetailView',
+  extends: 'foam.u2.View',
+  mixins: ['foam.u2.memento.Memorable'],
+
+  documentation: `Experimental new detail view that uses controller mode for a more seamless view/edit experience 
+  and offers more customisation while simplifying the code`,
+
+  axioms: [
+    foam.pattern.Faceted.create()
+  ],
+
+  requires: [
+    'foam.log.LogLevel',
+    'foam.u2.layout.Cols',
+    'foam.u2.layout.Rows',
+    'foam.u2.ControllerMode',
+    'foam.u2.dialog.Popup',
+    'foam.u2.stack.BreadcrumbView',
+    'foam.u2.stack.StackBlock'
+  ],
+
+  imports: [
+    'auth',
+    'ctrl',
+    'config? as importedConfig',
+    'currentMenu?',
+    'currentControllerMode?',
+    'setControllerMode?',
+    'stack?',
+    'translationService'
```

Comment:
Instead, add a property like this:
    {
      name: 'translationService',
      factory: function() {
        return this.__context__.translationService || foam.i18n.NullTranslationService.create(); }
    },

It will do the same thing when a translationService is available but not break when one isn't.